lib/webdriverio.js

```diff
@@ -487,6 +518,8 @@ let WebdriverIO = function (args, modifier) {
             commandGroup = prototype[namespace]
         }
 
+        rawCommands[name] = func
```

Comment:
We need a raw command to call it via `client.next`. If we call a wrapper than command will be added to the `commandList` twice. Also we don't need `saveScreenshot` anymore because we have a common code for taking screenshot